### DISCUSSION TITLE
How to block input event to lower layer UI?

### DISCUSSION THREAD

**Discussion by bwm0822** [rexrainbow/phaser3-rex-notes#468] [Q&A]
When two UI overlap, if mouse drag up down on upper layer UI, the lower layer UI also scroll up down.
I try to call setScrollerEnable() when upper layer UI shown, it works.
But is any way to block event to lower layer UI without call setScrollerEnable()?
(I use scrollablePanel)
![未命名](https://github.com/user-attachments/assets/690864fc-756e-41a2-a2fc-f9447d31bdc8)

**Comment by rexrainbow** [comment]
A possible solution is using modal dialog 
- [Demo](https://codepen.io/rexrainbow/pen/YzoaKOp)
- [Document](https://rexrainbow.github.io/phaser3-rex-notes/docs/site/ui-dialog/#modal)

**Reply by bwm0822** [reply]
I call modal({cover: {color: 0x0,alpha: 0.5,}}), it can block event to lower layer UI, but scroll is not working on upper layer UI.
If I call modal({cover: {color: 0x0,alpha: 0.5,}}).setInteractive(), scroll is working on upper layer UI, but it doesn't block event to lower layer UI.

**Comment by rexrainbow** [comment]
Open this thread as issue #469
